src/main/java/net/geforcemods/securitycraft/SCTags.java

```diff
@@ -42,11 +44,13 @@
 		public static final Tag<Item> REINFORCED_ACACIA_LOGS = tag("reinforced/acacia_logs");
 		public static final Tag<Item> REINFORCED_BIRCH_LOGS = tag("reinforced/birch_logs");
 		public static final Tag<Item> REINFORCED_CARPETS = tag("reinforced/carpets");
+		public static final Tag<Item> REINFORCED_COBBLESTONES = tag("reinforced/cobblestones");

```

Comment:
To be in line with Tags.Items.COBBLESTONE:
```suggestion
		public static final Tag<Item> REINFORCED_COBBLESTONE = tag("reinforced/cobblestone");
```